CONTRIBUTING.md

```diff
@@ -0,0 +1,71 @@
+# Contributing to AccessKit
+
+## Making a Pull Request
+
+### Cargo.lock File
+
+AccessKit intentionally includes the `Cargo.lock` file in our git repository.
+This is mainly due to our bindings to other languages.
```

Comment:
A better defense of why this is needed might be a good addition here. I don't know the reason myself so this was the best I had.